### ISSUE TITLE
Even before making any entry/selection for the required field screen reader says "invalid entry"

### ISSUE THREAD

**Issue by gkhadK** [vmware/clarity#4631]
**Clarity Version:** 3.1.1
**WCAG 2.1 Guideline:** 4.1.2: Name, Role, Value

**Test Environment:**
- Test URL: https://clarity.design/documentation/checkboxes
- OS: Windows 10
- User Agents/Assistive Technologies: Firefox 76.0
- Screen reader: NVDA 2020.1

**Steps to reproduce:**
1. Open the test URL (considering checkboxes example for describing the issue)
2. Go to section "Helper and error messages" and move focus on to the first checkbox using tab key and observe screen reader's focus.

**Expected result:**
Checkbox label should be announced along with it's control type and state.

**Actual result:**
- When checkbox is focused checkbox label is announced appropriately with state, but at the end of annoucement screen reader also say "invalid entry".

**Note:** Issue is applicable for other components as well, which are listed below:
- Similar   issue is observed for required datalist component like "template   validation" text input field. 
- Similar issue is observed for Select   component where "Invalid data" announcement made at the end when   user moves focus on the component.

**Comment by gnomeontherun** [comment]
Hi there, this is an automated message.

In an effort to dedupe and track accessibility reports consistently, this issue is being closed and tracked internally with our Accessibility team. It will still be triaged and addressed as part of ongoing work.
